tests/web/test_db_query.py

```diff
@@ -93,8 +108,107 @@ def test_apply_thread_quota(
     for rlimit in rate_limit_params:
         settings.add_rate_limit(rlimit)
     settings.set_resource_quota(resource_quota)
-    clickhouse_query_settings: MutableMapping[str, Any] = {}
+    clickhouse_query_settings: dict[str, Any] = {}
     _apply_thread_quota_to_clickhouse_query_settings(
         settings, clickhouse_query_settings, rate_limit_stats
     )
     assert clickhouse_query_settings == expected_query_settings
+
+
+def test_raw_query_success() -> None:
+    storage = get_storage(StorageKey("errors"))
```

Comment:
I would move all of this object building into a helper function(s) that can be called/overriden by both tests.